### DISCUSSION TITLE
Can I use changesets to publish to GitLab registry?

### DISCUSSION THREAD

**Discussion by runoncedev** [changesets/changesets#924] [Q&A] [answered]
Is it possible to use changesets to publish to custom registries? I.e. GitLab

**Comment by Andarist** [comment, marked as answer]
Yes. We delegate publishing to npm clients - so if you only call commands correctly, have correct values in files like `.npmrc` or configure your `package.json` correctly then it should work. If something doesn't work in relation to that then it would be classified as a bug.
